src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -194,7 +194,7 @@
     @Nullable
     private BuildingTownHall townHall;
     private int topCitizenId = 0;
-    private int maxCitizens  = Configurations.maxCitizens;
+    private int maxCitizens  = Configurations.gameplay.maxCitizens;
 
     private double overallHappiness = 5;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 5 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)